core/controller/src/main/resources/reference.conf

```diff
@@ -6,7 +6,7 @@ whisk {
     use-cluster-bootstrap: false
   }
   loadbalancer {
-    invoker-busy-threshold: 4
+    user-memory: 1024 m
```

Comment:
@cbickel Should this be named `invoker-user-memory`. I see following exception on startup as in my case `CONFIG_whisk_loadbalancer_invokerUserMemory` was not defined

```
Exception in thread "main" pureconfig.error.ConfigReaderException: Cannot convert configuration to a whisk.core.loadBalancer.ShardingContainerPoolBalancerConfig. Failures are:
  at 'whisk.loadbalancer':
    - Key not found: 'invoker-user-memory'.

	at pureconfig.package$.getResultOrThrow(package.scala:138)
	at pureconfig.package$.loadConfigOrThrow(package.scala:160)
	at whisk.core.loadBalancer.ShardingContainerPoolBalancer.<init>(ShardingContainerPoolBalancer.scala:159)
	at whisk.core.loadBalancer.ShardingContainerPoolBalancer$.instance(ShardingContainerPoolBalancer.scala:437)
	at whisk.core.controller.Controller.<init>(Controller.scala:117)
	at whisk.core.controller.Controller$.main(Controller.scala:258)
	at whisk.core.controller.Controller.main(Controller.scala)
```

Reply:
You are right.
I'll open a PR to correct this.
Thank you.

Reply:
#3993 has the fix.